### ISSUE TITLE
Questions about texture

### ISSUE THREAD

**Issue by MisEty** [3D-FRONT-FUTURE/3D-FRONT-ToolBox#27]
Thanks for your great work on this indoor scene dataset. I have downloaded the models and run scripts json2obj.py and I have some questions:
1. I noticed that most obj outputs are without textures? Why?
2. The outputs of json "17ff4014-6988-4b6e-9e3e-bf41b2cd9e05.json" have some textures, but floor and wall miss textures. How can I add textures?

**Comment by y-zheng18** [comment]
I have the same question. Have you found any solution @MisEty? Thanks!

**Comment by MisEty** [comment]
No, I can't find the textures. Finally I add my texture in blender instead

**Comment by learn2phoenix** [comment]
I seem to have cracked this. Basically, we need to read the `material` key in the mesh object. This corresponds to `uid` of the material in `data['material']`. After matching this, get the `jid` of the same material from `data['material']` and then this is basically the hash of the texture folder that you want to use with this mesh object. Let me know if it doesn't work for you.
